library/src/main/java/com/google/android/material/motion/MotionRuntime.java

```diff
@@ -60,4 +63,9 @@ public void next(T value) {
     Interaction<O, T> interaction, O target, ConstraintApplicator<T> constraints) {
     interaction.apply(this, target, constraints);
   }
+
+  /* Returns an ReactiveView */
+  public ReactiveView get(View v) {
+    return new ReactiveView(v);
```

Comment:
We want to avoid an object allocation every time we call `runtime.get()`, so let's cache this ReactiveView. You can do this by keeping a `SimpleArrayMap<View, ReactiveView>` inside of MotionRuntime.